Description: Relax build dependencies for Debian
Author: Arnaud Ferraris <aferraris@debian.org>
Forwarded: not-needed
Last-Update: 2024-07-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -48,7 +48,7 @@
 version = "0.5"
 
 [dependencies.heck]
-version = "0.5"
+version = "0.4"
 
 [dependencies.proc-macro-error]
 version = "1.0"
@@ -57,7 +57,7 @@
 version = "1.0"
 
 [dependencies.quick-xml]
-version = "0.37"
+version = ">= 0.36, < 1.0"
 features = ["serialize"]
 
 [dependencies.quote]
@@ -71,7 +71,7 @@
 version = "2.0"
 
 [dev-dependencies.gio]
-version = "0.20"
+version = "0.21"
 
 [dev-dependencies.serial_test]
-version = "3.2"
+version = "2.0"
